src/huntsman/pocs/core.py

```diff
@@ -59,7 +59,7 @@ def after_state(self, event_data):
         """
         self.say(f"Finished with the {self.state} state. The next state is {self.next_state}.")
 
-    # Private methods
+    # private methods
```

Comment:
Just curious as to why this change? I have been commenting with capitals. Not important.

Reply:
umm not sure, probably some auto formatting from my editor. Definitely not an intentional change